.github/issue-templates/release.md

```diff
@@ -1,16 +1,16 @@
 # Release {version}
 We are releasing a new version, this issue will keep track of the progress between the first release candidate (20th of each month) to the final release (27th of each month).
 
-After the 20th of each month we start the release process that ends 7 days after, during that period we enter a Feature Freeze. While in the Feature Freeze, we will only be merging bug fixes and not new features.
+Beginning on the 20th of each month, we will start the release process which ends 7 days later (the 27th). During this period of time, we will enter a "Feature Freeze". This Feature Freeze means that we will only be merging pull requests which fix bugs and not ones which add new features.
 
-For any regression, open a new issue and link to this one.
+When you find a bug that is a regression, please open a new issue and link it to this one.
 
 
 ## Before Release - Preparation - 1 business day before the day 20th
 - [x] Create the issue to track the release progress
-- [ ] Define the highlights from release PRs as suggestion to be included on Blog Post
-- [ ] Talk to the Marketing Team about the Blog Post release
-- [ ] Talk to the Documentation Team about the Docs release
+- [ ] Define the highlights from release PRs as suggestion to be included on Blog Post (as a new comment to this issue with links to the PRs)
```

Comment:
Hasn't this actually been done on the website repo the past couple of releases?

Reply:
I wasn't aware of that. @rodrigok can you comment here?

Reply:
https://github.com/RocketChat/rocketchat.github.io/issues/162 this is what has been happening

Reply:
@graywolf336 @geekgonecrazy Yes, we are opening a new issue on the website repo to list and discuss the highlights, we should edit a put the link here to be more clear